build/build_defs.bzl

```diff
@@ -30,23 +30,4 @@ def go_yacc(src, out, visibility=None):
       visibility = visibility,
       local = 1,
   )
-
-def genfile_check_test(src, gen):
```

Comment:
this may be used in other places, so don't delete this function just yet

Reply:
Done